docs/standard-library/vectorized-stl-algorithms.md

```diff
@@ -0,0 +1,92 @@
+---
+description: "Vectorized STL Algorithms"
+title: "Vectorized STL Algorithms"
+ms.date: "09/19/2025"
+helpviewer_keywords: ["Vector Algorithms", "Vectorization", "SIMD"]
+---
+# Vectorized STL Algorithms
+
+Under certain conditions, STL algorithms execute not element-wise, but multiple element at once on a single CPU core. This is possible due to SIMD (single instruction, multiple data). The use of such approach instead of 
+element-wise approach is called vectorization. An implementation that is not vectorized is called scalar.
```

Comment:
```suggestion
Under certain conditions, STL algorithms execute not element-wise, but multiple element at once on a single CPU core. This is possible due to SIMD (single instruction, multiple data). The use of such approach instead of element-wise approach is called vectorization. An implementation that is not vectorized is called scalar.
```

Unnecessary wrapping here.